src/test/java/com/example/universe/simulator/entityservice/integration/GalaxyIntegrationTest.java

```diff
@@ -63,14 +64,21 @@ void test() throws Exception {
         resultList = objectMapper.readValue(response.getContentAsString(), new TypeReference<>() {});
         assertThat(resultList.getContent()).hasSize(2);
 
-        // -----------------------------------should return entity-----------------------------------
+        // -----------------------------------should return entity and store it in cache-----------------------------------
 
         // when
         response = performRequest(get("/galaxy/get/{id}", addedDto1.getId()));
         // then
         GalaxyDto resultDto = objectMapper.readValue(response.getContentAsString(), GalaxyDto.class);
         assertThat(resultDto).isEqualTo(addedDto1);
 
+        //check if resultDto got into cache
+        assertThat(redisTemplate.hasKey("Galaxy::" + addedDto1.getId())).isTrue();
+        GalaxyDto fromCache = modelMapper.map(
+                objectMapper.convertValue(redisTemplate.opsForValue().get("Galaxy::" + addedDto1.getId()), Galaxy.class),
```

Comment:
`redisTemplate.opsForValue().get("Galaxy::" + addedDto1.getId())` is already Galaxy type, no need for excessive conversion

Reply:
redisTemplate.opsForValue().get("Galaxy::" + addedDto1.getId())
Yes it is Galaxy type, but if you look closely, I needed to compare it to GalaxyDto type. It is impossible to compare those objects without conversion.

Reply:
I mean `objectMapper.convertValue` is not necessary, conversion to GalaxyDto should happen, no doubt .

Reply:
Okay, now I see what you meant. Will modify